src/main/java/attendance/model/AttendancesFile.java

```diff
@@ -0,0 +1,74 @@
+package attendance.model;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+
+public class AttendancesFile {
+
+    private static final String CSV_HEADER_FORMAT = "nickname,datetime";
+    private static final String COLUMN_DELIMITER = ",";
+    private static final int EXPECTED_COLUMN_COUNT = CSV_HEADER_FORMAT.split(COLUMN_DELIMITER).length;
+    private static final int NICKNAME_INDEX = 0;
+    private static final int ATTENDANCE_DATE_TIME_INDEX = 1;
+    private static final String DATETIME_FORMAT_PATTERN = "yyyy-MM-dd HH:mm";
```

Comment:
DateTimeFormatter 는 생성 비용이 비싼 객체이기 때문에 보통 한 번 정의해두고 재사용하는 방식을 많이 사용합니다 🙂 

```suggestion
    private static final DateTimeFormatter DATETIME_FORMAT_PATTERN = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```

Reply:
`DateTimeFormatter`는 단순한 포맷터가 아니라, 
출력/파싱, 지역화, 연대/타임존 변환, 필드 해석 등 다양한 기능을 수행하네요.

객체 생성 시 이러한 요소들을 조합하는 과정에서 객체 검증, 복잡한 설정 처리, 불변성 유지 등의 비용이 발생하며,
동일한 포맷을 여러 번 사용할 경우 매번 생성하지 말고, 재사용하는 것이 성능 측면에서 확실히 유리하겠네요 👍 👍 

감사합니다! 반영하겠습니다!